examples/tv-app/android/java/AppImpl.h

```diff
@@ -86,9 +86,10 @@ class DLL_EXPORT ContentAppImpl : public ContentApp
                    const char * szApplicationVersion, const char * setupPIN, jobject manager) :
         mApplicationBasicDelegate(kCatalogVendorId, BuildAppId(vendorId), szVendorName, vendorId, szApplicationName, productId,
                                   szApplicationVersion),
-        mAccountLoginDelegate(setupPIN), mContentLauncherDelegate(ContentAppCommandDelegate(manager), { "image/*", "video/*" },
-                                                                  to_underlying(SupportedStreamingProtocol::kDash) |
-                                                                      to_underlying(SupportedStreamingProtocol::kHls)),
+        mAccountLoginDelegate(setupPIN),
+        mContentLauncherDelegate(new ContentAppAttributeDelegate(manager), { "image/*", "video/*" },
```

Comment:
Where is the object that is allocated here via `new` freed?

Reply:
Dont think I need to create object here using new. Removed.